utils-testing-coroutines/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+
+dependencies {
+    implementation "junit:junit:$versions.junit"
```

Comment:
Should this not be `testImplementation` ? Or is it because this library is only meant to be consumed as a `testImplementation` dependency?

Reply:
This can be removed completely as there are no tests.